app/components/Mods/AlphaSwitch.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Switch, Tooltip, FormLabel } from '@material-ui/core';
+import { useSetRecoilState, useRecoilValue } from 'recoil';
+import { alphaState } from '../../store';
+
+const AlphaSwitch: React.FunctionComponent = () => {
```

Comment:
So, sorry for changing my mind, but I think this switch shouldn't go here after all. A tiny, tiny fraction of people using mods care about the alpha (from what I can tell in the website stats), so I don't think the button should be so prominent.

I'd prefer if this switch is moved to the settings tab. It can replace the "Show Alpha Settings" switch. So you can go to the settings tab, enable "View Mods for Outer Wilds Alpha", which simultanesly expands the alpha settings and switches to alpha mode.